uber/automated_emails.py

```diff
@@ -19,7 +19,7 @@ class AutomatedEmail:
 
     def __init__(self, model, subject, template, filter, *, when=(),
                  sender=None, extra_data=None, cc=None, bcc=None,
-                 post_con=False, needs_approval=True, ident=None):
+                 post_con=False, needs_approval=True, ident=None, allow_during_con=False):
```

Comment:
By setting this to false by default, we keep this backwards-compatible with how things were before.